ShopAppTests/App/Base/Views/BasePickerSpec.swift

```diff
@@ -0,0 +1,144 @@
+//
+//  BasePickerSpec.swift
+//  ShopAppTests
+//
+//  Created by Radyslav Krechet on 2/21/18.
+//  Copyright © 2018 RubyGarage. All rights reserved.
+//
+
+import Nimble
+import Quick
+
+@testable import ShopApp
+
+class BasePickerSpec: QuickSpec {
+    override func spec() {
+        var view: BasePicker!
+        var textField: UITextField!
+        var placeholderLabel: UILabel!
+        var underlineView: UIView!
+        
+        beforeEach {
+            view = BasePicker()
+            
+            textField = self.findView(withAccessibilityLabel: "text", in: view) as! UITextField
+            placeholderLabel = self.findView(withAccessibilityLabel: "placeholder", in: view) as! UILabel
+            underlineView = self.findView(withAccessibilityLabel: "underline", in: view)
+        }
+        
+        describe("when view initialized") {
+            it("should have correct picker view") {
+                expect(view.pickerView.backgroundColor) == .white
```

Comment:
Change `==` to `to(equal(.white))`

Reply:
Why? It's the same comparison. Let's you short form if it possible.